### DISCUSSION TITLE
Quick preview of issues from the project (Board/Table)

### DISCUSSION THREAD

**Discussion by avishwakarma** [community/community#12806] [Projects and Issues]
Trying Projects on Github for quite some time, and I love it where it's going.

Would have been great if we can preview the issue inside the project view (Board/Table) on a sidebar or panel.

**Comment by aboterolh** [comment]
You can, so to speak, before converting them to Issues. After that, definitely need that preview feature.

**Reply by avishwakarma** [reply]
I was asking about issues specifically.

**Reply by rodriin** [reply]
Agreed, this feature is missing for anyone trying to use the board view in the new Projects Beta. It's cumbersome to have to navigate tabs just to see a little of the description for an issue or to assign someone to the issue, add a label, etc.

**Reply by aboterolh** [reply]
I had not used the **non-Beta** Projects, but I checked one, and they do have this Preview. Only Beta ones do not have it 🙃
